### PR TITLE
Adds daily deploy guidelines

### PR DESCRIPTION
**Pull request by joaopslins**

### CODE REVIEW

File: checklists/daily_deploy.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Process Setup
+- Create a Release tag on Github to tag the release PRs;
+- Create a PR template for Releases -- They should have a different one from other PRs;
+- Have tags to categorize PRs: Feature, Fix, Tech Debt, etc;
+- Have a #release channel on Slack, this channel will be used to inform the clients about the releases;
+
+## Step by Step
+
+### Pre-deploy
+1. Make sure the release branch is created, if not create it from master. Use `release-yyyy-mm-dd.x` as branch name, where X is the number of the deploy on the same day -- usually X is zero;
```
Comment (laisvarejao) [inline checklists/daily_deploy.md]: small change suggestion: "X starts at zero;"

File: checklists/daily_deploy.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Deploy
+1. Check for status of relevant services -- CircleCI, Heroku, etc;
+2. Merge the PR and watch CI to make sure the build and deploy workflow runs smoothly;
+3. Make sure all the tasks ran successfully, like migrations;
+
+### Post-deploy
+1. After the deploy is done, test all the deployed tasks on production;
+2. Create the Github Release, and include the Hotfix PRs as well as the Merged PRs, and link the Release PR. The release branch must be the production;
+3. Message the #release channel on Slack with all the merged PRs, with a link to the corresponding Github Release. Do not include PR IDs in this list;
```
Comment (laisvarejao) [inline checklists/daily_deploy.md]: Perhaps set an example for this?

File: checklists/daily_deploy.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+4. Make sure everything that needs documentation has the link to the documentation attached to the Asana card -- This is an effort to make sure your documentation is up to date with your application;
```
Comment (laisvarejao) [inline checklists/daily_deploy.md]: Should we clarify what type of things may need documentation? Perhaps examples.

File: checklists/daily_deploy.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+5. Move the cards from in the Sprint board @ Asana to Done. You may move to other columns depending on your needs, like `Success`, where some other person / team should know about this card being live.
```
Comment (laisvarejao) [inline checklists/daily_deploy.md]: Typo: "from the Sprint board..."

File: guidelines.md [hunk 5]
```diff
@@ -219,6 +220,19 @@ There are different techniques that can be used such as **Daki Exercise**, **Sto
 - What was bad
 - What can improve
 
+### Deployment
+
+It is proven that a high deploy frequency is a good DevOps practice and gives a lot of benefits in the Agile process. Google considers multiple deploys per day as a [factor to consider a team as an Elite Performer](https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling.).
```
Comment (laisvarejao) [inline guidelines.md]: I would maybe rephrase this first sentence:

"It is proven that a high deploy frequency is a good DevOps practice and improves productivity of Agile teams."

File: guidelines.md [hunk 6]
```diff
@@ -219,6 +220,19 @@ There are different techniques that can be used such as **Daki Exercise**, **Sto
...
+
+Deploying more frequently:
+ - Represents having smaller changes, which means less risk.
+ - Decreases the time between a feature being ready and being live.
+ - Encourages better feature QA, since features will spend less time on Staging until they launch to Production.
+ - Increases code review quality, since changes tend to be smaller.
+ - Incentivizes breaking up bigger tasks in smaller ones that can be deployed as soon as they are ready.
```
Comment (laisvarejao) [inline guidelines.md]: More benefits:

- Facilitates bugs detection.
- ... not sure how to phrase it here, but something that mentions it's best for the team, as they don't have to meet a single deploy day deadline.

File: checklists/daily_deploy.md (hunk 2 above)
Comment (joaopslins) [reply checklists/daily_deploy.md]: It should be the same list as the one in the Release -- 3rd step in Pre-deploy. But without the PR IDs `(#1234)`. Do you need this should be better explained?
